scripts/cron-test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -vx
+
+# Revert the cache-min setting, since the internal cache does not apply to
+# these repos (and causes problems in lookups)
+npm config set cache-min 10
+
+# Use newer, faster npm
+npm install -g npm@4.0.2
+
+if ! npm install; then
+  echo "npm install failed! Exiting..."
+  exit ${FAILED_SETUP}
+fi
+
+npm install
+npm run test:unit
+
+# Don't run integration tests if node version < 8
+# Due to async/await being used in ITs, we can't run ITs on node version < 8
+echo "Checking node version..."
```

Comment:
Since we're running these tests internally, we should be able to use node 8 and run these ITs. See how we did this with the Widget, in [`setup.sh`](https://github.com/okta/okta-signin-widget/blob/master/scripts/setup.sh#L8).

Reply:
The node versions are set in `.travis.yml` right?
```
node_js:
- '4'
- '6'
- '8'
```
Is it fine to use `setup_service` to override node version for each run? I believe travis will run the tests once for each node version specified in `.travis.yml`.

Reply:
Ah - sorry I misunderstood what was going on earlier. I thought the `cron-test.sh` file was used for our Bacon runs.

`setup_service` will only work on our containers, so we'll need to either change our `.travis.yml` file to run on _only_ Node 8, or have separate logic for Bacon support in the future.

@robertjd - Coming back to the earlier question Vijet proposed, do we want this CRON job to run only for Node 8?